plots/laws/thermodynamics/quantum_statistics/fermi_dirac_energy_dependence.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+from sympy import symbols, Eq, solve
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import print_expression, units
+from symplyphysics.laws.thermodynamics.fermi_dirac_statistics import (
+    single_particle_state_distribution as distribution_law,
+)
+
+occupancy = distribution_law.occupancy_of_state
+energy = distribution_law.energy_of_state
+chemical_potential = distribution_law.total_chemical_potential
+temperature = distribution_law.temperature
+
+reduced_energy = symbols("reduced_energy")
+reduced_energy_eqn = Eq(reduced_energy, energy / chemical_potential)
```

Comment:
Do we have a law for it?

Reply:
Same as in the temperature plot. This is only used once here, for the sake of simplification.